### PR TITLE
Completed Postgres CRUD API requests, made requested changes, front end working

### PR DESCRIPTION
**Pull request by uabrami**
Hi Andrew and Aysun,

Please review my latest changes.

Changes made:
1) Made all requested changes from last pull request
2) Completed Postgres CRUD API requests
3) Adjusted front end to work with new data set, front end working

NOTES: 
-I made an config file for my postgres in preparation for deployment, but commented out and am using local db for testing at the moment. Should be easy to switch for deployment.
-Pushing up my Cassandra files for review, after review, hope to add to my git ignore.

Thank you,
Uma

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -37,6 +35,31 @@ app.post('/actors/add', (req, res) => {
...
+
+app.delete('/actors/delete', (req, res) => {
+  const {id} = req.body;
+  console.log(id);
+  db.deleteActor(id, (err, results) => {
+    if (err) {
+      res.sendStatus(500);
+      console.log(`actors DELETE error=${err}`);
+    }
+    res.send(`User deleted for ID: ${id}`);
```
Comment (ahchung23) [inline server/index.js:59]: Do you think it would be better to send an actual data object for more clarity of what was deleted or updated?

**Comment by asuluntas** [comment]
Sorry that comes after we talked about it but  don't forget to send status code even the request is successful. And I' not commenting on delete because we will refactor it.
Looks great!

File: server/index.js [hunk 2]
```diff
@@ -37,6 +35,31 @@ app.post('/actors/add', (req, res) => {
...
+  db.updateActor(name, title, role, photo, bio, filmography, id, (err, results) => {
+    if (err) {
+      res.sendStatus(500);
+      console.log(`actors PUT error=${err}`);
+    }
+    res.send(`User updated for ID: ${id}`);
+  });
+});
+
+app.delete('/actors/delete', (req, res) => {
```
Comment (ahchung23) [inline server/index.js:51]: Remember to delete from both tables if possible to reduce error :)

File: server/index.js [hunk 3]
```diff
@@ -37,6 +35,31 @@ app.post('/actors/add', (req, res) => {
   });
 });
 
+app.put('/actors/update', (req, res) => {
+  const {name, title, role, photo, bio, filmography, id} = req.body;
+  console.log(req.params.id);
+  console.log('PUT', name, title, role, photo, bio, filmography, id);
+  db.updateActor(name, title, role, photo, bio, filmography, id, (err, results) => {
```
Comment (ahchung23) [inline server/index.js:42]: Also two different types of update may be possible. Updating Actors and updating actor information for a movie!

**Review by ahchung23 — COMMENTED** [review]
Looks good Uma!

I didn't see any problems with the seeding section of the repo.
I would be careful about how to design some of the CRUD requests so I commented on them as we discussed in our zoom call!

Let's get this to deployment!!! 👍

**Comment by uabrami** [comment]
@asuluntas @ahchung23 Thank you both for your review and comments, I will incorporate based on our last conversation. I will now merge into my master. Best, Uma

**Pull request merged**